### ISSUE TITLE
Crashes with json decoding error on systems with NVMe drives when using Docker image.

### ISSUE THREAD

**Issue by nrhtr** [packethost/packet-hardware#21]
Hi,

I get this error when attempting to run packet-hardware from the provided Dockerfile on systems with NVMe drives.

```
cmd="smartctl --all --json /dev/nvme0n1", method="cmd_output", klass="utils"
cmd="smartctl", method="cmd_output", errorcode="1", klass="utils"
Traceback (most recent call last):
  File "/usr/local/bin/packet-hardware", line 9, in <module>
    load_entry_point('packet-hardware==1.3', 'console_scripts', 'packet-hardware')()
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/packethardware/inventory.py", line 67, in inventory
    components.extend(eval(t).list(lshw))
  File "/usr/local/lib/python3.5/dist-packages/packethardware/component/disk.py", line 16, in list
    disks.append(cls(disk))
  File "/usr/local/lib/python3.5/dist-packages/packethardware/component/disk.py", line 29, in __init__
    self.data["smart"] = utils.get_nvme_attributes(self.lsblk["name"])
  File "/usr/local/lib/python3.5/dist-packages/packethardware/utils.py", line 124, in get_nvme_attributes
    return json.loads(smartctl_json)["nvme_smart_health_information_log"]
  File "/usr/lib/python3.5/json/__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.5/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.5/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

It appears to be due to smartctl lacking JSON support in the Ubuntu image used. I fixed this for myself by using Ubuntu 20.04

```
localhost:~# docker run --rm -it --privileged --net=host -v /dev:/dev --entrypoint="" private-docker-hub.net/packet-hardware:master smartctl --all --json /dev/nvme0n1
smartctl 6.5 2016-01-24 r4214 [x86_64-linux-5.4.72-0-lts] (local build)
Copyright (C) 2002-16, Bruce Allen, Christian Franke, www.smartmontools.org

=======> UNRECOGNIZED OPTION: json

Use smartctl -h to get a usage summary
```

**Comment by dustinmiller** [comment]
Yeah, the --json option appeared in smartmontools around version 7

**Comment by dustinmiller** [comment]
@nrhtr want to give the latest version a whirl and see it's any better?

**Comment by nrhtr** [comment]
@dustinmiller Thanks, that works fine (once https://github.com/packethost/packet-hardware/pull/23 is merged)

**Comment by dustinmiller** [comment]
Merged #23
